pkg/bloombuild/builder/builder_test.go

```diff
@@ -26,6 +26,7 @@ import (
 
 func Test_BuilderLoop(t *testing.T) {
 	logger := log.NewNopLogger()
+	//logger := log.NewLogfmtLogger(os.Stdout)
```

Comment:
Could be removed I assume

Reply:
I'm leaving this around for debugging if necessary but happy to remove them if you feel otherwise